senza/templates/postgresapp.py

```diff
@@ -24,6 +24,8 @@
     - ImageVersion:
         Description: "Docker image version of spilo."
   {{/docker_image}}
+  Tags:
+    - SpiloCluster: "{{=<% %>=}}{{Arguments.version}}<%={{ }}=%>"
```

Comment:
tags have no order, so you should use a simple dict (instead of a list of dicts)